pkg/acsengine/json.go

```diff
@@ -28,6 +28,22 @@ func PrettyPrintArmTemplate(template string) (string, error) {
 	return template, nil
 }
 
+// PrettyPrintArmParameters will pretty print the arm template ensuring the header is $schema and contentVersion are at top.
+func PrettyPrintArmParameters(template string) (string, error) {
+	translateParams := [][]string{
+		{"\"contentVersion\"", "\"$zcontentVersion\""},
+	}
+
+	template = translateJSON(template, translateParams, false)
+	var err error
```

Comment:
Because you're not re-using the error reference `err` anywhere other than the if block below, you don't have to declare it here; `if template, err := PrettyPrintJSON...` is probably more idiomatic.